core/order.go

```diff
@@ -1305,7 +1305,9 @@ func (n *OpenBazaarNode) getMarketPriceInSatoshis(pricingCurrency, currencyCode
 		pricingDef, pErr  = n.LookupCurrency(pricingCurrency)
```

Comment:
What is the point of the currency lookups in this function? The only way they're used is to compare whether the codes of each cur def are equal, but can't that just be done by comparing the pricingCurrency and currencyCode arguments?

Reply:
Mostly to take advantage of `currencyDef.Equal(pricingDef)` and supporting the eventuality that we will be passing around CurrencyValues (with variable divisibilities) throughout instead of the raw lookup code. `Equal` not only accounts for the code being normalized and matching, but also the divisibilities of each currency matching.